tools/wasm-sourcemap.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
```

Comment:
Add copyright header

Reply:
I can find one in the existing tools. What's the license for new contributions?

Reply:
Sorry I guess we don't do that for emscripten... we probably should.. but no need to change that in this PR.